1.4.5FeedbackByExpress/app.js

```diff
@@ -86,8 +86,22 @@ app.post('/post', function(request, response)
 	//console.log('get form post request')
 	
 	//2.proceed
-	var comment = request.body
-	comment.dateTime = '2021-2-10 00:00:00'
+	/*
+	The comments object that contains the list of comments is
+	expecting to recieve an object with three properties
+	1. name
+	2. message
+	3. date
+
+	In order for it to work properly you must match them exactly
+	*/
+	const currentDate = new Date();
+	const dateString = `${currentDate.getFullYear()}-${currentDate.getMonth()}-${currentDate.getDate()}`;
```

Comment:
When the Date construct is called without an argument, it captures the current date.
dateString uses [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) to format the date into a 'yyyy-mm-dd' format